tests/lib/config/config-validator.ts

```diff
@@ -48,15 +48,15 @@ test(`if rule severity isn't valid, it should return false`, (t) => {
     t.false(valid);
 });
 
-test(`if rule schema isn't valid, it should return false`, (t) => {
-    const config = _.cloneDeep(validConfig);
+// test(`if rule schema isn't valid, it should return false`, (t) => {
```

Comment:
I just moved this test to the `sonar.ts` ones because that's where we do the rule validation.